components/content_settings/renderer/brave_content_settings_agent_impl.cc

```diff
@@ -224,7 +224,15 @@ bool BraveContentSettingsAgentImpl::AllowStorageAccessSync(
 bool BraveContentSettingsAgentImpl::AllowScriptFromSource(
     bool enabled_per_settings,
     const blink::WebURL& script_url) {
-  const GURL secondary_url(script_url);
+  GURL secondary_url(script_url);
+  // For scripts w/o sources it should report the domain / site used for
+  // executing the frame (which most, but not all, of the time will just be from
+  // document.location
```

Comment:
Uber-nit, here: you forgot the closing `)` in the source comment, after `// document.location)`